cli/lint-tree.js

```diff
@@ -27,20 +27,25 @@ export default class LintTree extends Filter {
   processString(content, relativePath) {
     let report = this.cli.executeOnText(content, path.join(this.rootDir, relativePath));
     let result = report.results[0] || {};
-    let errors = result.messages || [];
+    let messages = result.messages || [];
 
-    errors = errors.filter((error) => !IGNORED_FILE_MESSAGE_REGEXP.test(error.message));
+    messages = messages.filter((msg) => !IGNORED_FILE_MESSAGE_REGEXP.test(msg.message));
 
     if (this.generateTests) {
-      return this.testGenerator(relativePath, errors);
+      return this.testGenerator(relativePath, messages);
+    }
+
+    if (messages.length) {
```

Comment:
Would you mind changing this to `if (messages.length > 0)`? I prefer to make these kinds of conditionals explicit rather than relying on type coercing `0` to `false`.